crates/pallet-domains/src/staking_epoch.rs

```diff
@@ -88,6 +75,60 @@ pub(crate) fn do_unlock_pending_withdrawals<T: Config>(
     Ok(())
 }
 
+/// Operator takes `NominationTax` of the current epoch rewards and stake them.
+pub(crate) fn operator_take_reward_tax_and_stake<T: Config>(
+    domain_id: DomainId,
+) -> Result<(), Error> {
+    DomainStakingSummary::<T>::try_mutate(domain_id, |maybe_domain_stake_summary| {
+        let stake_summary = maybe_domain_stake_summary
+            .as_mut()
+            .ok_or(TransitionError::DomainNotInitialized)?;
+
+        while let Some((operator_id, reward)) = stake_summary.current_epoch_rewards.pop_first() {
+            Operators::<T>::try_mutate(operator_id, |maybe_operator| {
+                let operator = maybe_operator
+                    .as_mut()
+                    .ok_or(TransitionError::UnknownOperator)?;
+
+                // calculate operator tax, mint the balance, and stake them
+                let operator_tax = operator.nomination_tax.mul_ceil(reward);
```

Comment:
Shouldn't this be floor?

Reply:
Yeah, I was skeptic about this. Thanks for bringing up!

TBH, I don't have a strong opinion on this. So far, we always rounded down with `mul_floor` and preferred dust to go to treasury. 

In this case, dust goes to operator instead of the pool reward. If we prefer dust to go to treasury directly or indirectly, then `floor_mul` make sense here too